src/steps/codeownersComment.js

```diff
@@ -171,6 +171,16 @@ export default async function codeownersComment ({
     return null
   }
 
+  // Skip if no teams/groups are assigned (only individuals)
```

Comment:
Can we make it: "Skip if sec team is not assigned"? to be stricter and since the goal of this action is to be sec-team focused.